src/app/beer_garden/db/mongo/pruner.py

```diff
@@ -51,6 +63,8 @@ def run(self):
                 )
                 task["collection"].objects(query).no_cache().delete()
 
+            MongoPruner.cancel_in_progress(self)
```

Comment:
This looks weird.  I'm assuming this is supposed to be `self.cancel_in_progress()`?

Also, worth mentioning is that by convention, if a method is intended for internal use only (you don't want / expect it to be called outside of the MongoPruner's own code), it should start with an underscore.  If that's the case (looks like it based on the usage here) and taking into account my suggested name elsewhere, then this should possibly be:

```python
self._cancel_outstanding_requests()
```